shared-helpers/src/types/backend-swagger.ts

```diff
@@ -1828,6 +1828,30 @@ export class AuthService {
   }
 }
 
+export class MapLayersService {
+  /**
+   * List map layers
+   */
+  list(
+    params: {
+      /**  */
+      jurisdictionId?: string
+    } = {} as any,
+    options: IRequestOptions = {}
+  ): Promise<any> {
```

Comment:
I think this returning `any` indicates something is up with the controller type definition

or maybe the client generation needs to be re-run?

Reply:
I just needed to re-run!